package.json

```diff
@@ -24,6 +24,7 @@
     "cordova-js": "^4.2.2",
     "cordova-serve": "^2.0.0",
     "dep-graph": "1.1.0",
+    "dependency-ls": "^1.1.1",
```

Comment:
this and all the other dependencies that are readded are not reflected in the PR title and description